src/sentry/eventstore/models.py

```diff
@@ -439,6 +439,18 @@ def _hashes_from_sorted_grouping_variants(variants):
 
         return filtered_hashes, tree_labels
 
+    def normalize_stacktraces_for_grouping(self, grouping_config):
+        """Normalize stacktraces and clear memoized interfaces
+
+        See stand-alone function normalize_stacktraces_for_grouping
+        """
+        from sentry.stacktraces.processing import normalize_stacktraces_for_grouping
+
+        normalize_stacktraces_for_grouping(self.data, grouping_config)
+
+        # We have modified event data, so any cached interfaces have to be reset:
+        vars(self).pop("interfaces", None)
```

Comment:
Presumably `del self.interfaces` in a try/except does not work because of descriptors, but maybe we can turn this at least into `self.__dict__.pop("interfaces", None)` which is a bit less confusing.

Reply:
Done.